common/tests/fixtures/cardano/sign_tx.json

```diff
@@ -2400,6 +2758,64 @@
         ]
       }
     },
+    {
+      "description": "Ordinary transaction with a required signer and 258-tagged sets",
```

Comment:
I noticed that required signers with 258 tag are covered here and also in Plutus tx tests, but certificates with 258 are not covered at all. Maybe we could remove this test (and leave this case for the Plutus test) and add a certificate + 258 tag test instead?

Reply:
You're right, I missed certificates, makes sense to add to `sign_tx.json` and keep the Plutus one, that way all possible occurences of tagged sets should be covered